README.md

```diff
@@ -1,26 +1,85 @@
-# <img src=	https://upload.wikimedia.org/wikipedia/commons/7/7a/ROS_cat.png width="250" height="140">
+<div class="row">
+  <div class="column">
 
+  </div>
+  <div class="column">
+
+  </div>
+</div>
+<p float="left">
+    <a href="https://revolve.no/"><img align=left src="https://raw.githubusercontent.com/RevolveNTNU/RosNet/86-improve-readme/.github/main/revolve.svg" width="30%" height="100"/></a>
+    <a href="https://ros.org/"><img align=right margin src="https://raw.githubusercontent.com/RevolveNTNU/RosNet/86-improve-readme/.github/main/ros.png" height="100"/></a>
+</p>
+<br>
```

Comment:
One `<br>` should be enough?

Reply:
The badges and title will come inbetween the pictures if not